shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -160,11 +160,21 @@
   frame_size_ = frame_size;
 }
 
-void FlutterPlatformViewsController::PrerollCompositeEmbeddedView(int view_id) {
+void FlutterPlatformViewsController::PrerollCompositeEmbeddedView(
+    int view_id,
+    std::unique_ptr<EmbeddedViewParams> params) {
   picture_recorders_[view_id] = std::make_unique<SkPictureRecorder>();
   picture_recorders_[view_id]->beginRecording(SkRect::Make(frame_size_));
   picture_recorders_[view_id]->getRecordingCanvas()->clear(SK_ColorTRANSPARENT);
   composition_order_.push_back(view_id);
+
+  if (current_composition_params_.count(view_id) == 1 &&
+      current_composition_params_[view_id] == *params.get()) {
+    // Do nothing if the params didn't change.
+    return;
+  }
+  current_composition_params_[view_id] = EmbeddedViewParams(*params.get());
+  views_need_recomposite.insert(view_id);
```

Comment:
nit: better name would be `view_needs_recomposition_`

Reply:
done, kept is as `views_need_recomposition_` since it is a set.

Reply:
Maybe `views_to_recomposite_`?

Reply:
done